020_Distributed_Cluster/20_Add_failover.asciidoc

```diff
@@ -12,11 +12,12 @@ in exactly the same way as you started the first one (see
 <<running-elasticsearch>>), and from the same directory. Multiple nodes can
 share the same directory.
 
-As long as the second node has the same `cluster.name` as the first node (see
-the `./config/elasticsearch.yml` file), it should automatically discover and
-join the cluster run by the first node. If it doesn't, check the logs to find
-out what went wrong.  It may be that multicast is disabled on your network, or
-that a firewall is preventing your nodes from communicating.
+When you run a second node on the same machine, it automatically discovers 
+and joins the cluster as long as it has the same `cluster.name` as the first node (see
+the `./config/elasticsearch.yml` file). However, for nodes running on different machines
```

Comment:
If we add the link to the config section above, we can probably remove the parenthesized tip.